scripts/clib/clib_test_all.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
shell is good enough, but if possible, can this script be a python script? Practically, we usually struggle with windows tests because its different from unix.
But running bash script on windows is not easy.

Reply:
plus maintainability. Sure we could get issues fixed but if possible we should limit the # of languages we use (as much as possible).

Reply:
I think that could be doable, as python supports file IO system calls, and I think it should also be possible to use os.system() to run cargo for building crates.

One question, would that python script be run in RustPython, or just python?

Reply:
Just python will be ok.

Reply:
> Just python will be ok. We have to fix rustpython to run any python script, eventually.

Done!